src/NewsService.Models.Db/DbNews.cs

```diff
@@ -1,26 +1,57 @@
-﻿using System;
-using System.ComponentModel.DataAnnotations;
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
 
 namespace LT.DigitalOffice.NewsService.Models.Db
 {
     public class DbNews
     {
-        [Key]
+        public const string TableName = "News";
+
         public Guid Id { get; set; }
-        [Required]
         public string Content { get; set; }
-        [Required]
         public string Subject { get; set; }
         public string Pseudonym { get; set; }
-        [Required]
         public Guid AuthorId { get; set; }
-        [Required]
         public Guid SenderId { get; set; }
-        [Required]
         public DateTime CreatedAt { get; set; }
-        [Required]
-        public Guid DepartmentId { get; set; }
-        [Required]
+        public Guid? DepartmentId { get; set; }
         public bool IsActive { get; set; }
     }
-}
\ No newline at end of file
+
+    public class DbNewsConfiguration : IEntityTypeConfiguration<DbNews>
+    {
+        public void Configure(EntityTypeBuilder<DbNews> builder)
+        {
+            builder.
+                ToTable(DbNews.TableName);
+
+            builder.
+                HasKey(p => p.Id);
+
+            builder
+                .Property(p => p.CreatedAt)
+                .IsRequired();
+
+            builder
+                .Property(p => p.Content)
+                .IsRequired();
+
+            builder
+                .Property(p => p.Subject)
+                .IsRequired();
+
+            builder
+                .Property(p => p.AuthorId)
+                .IsRequired();
+
+            builder
+                .Property(p => p.SenderId)
+                .IsRequired();
+
+            builder
```

Comment:
You can don't write this, because bool can't be null

Reply:
same with other valuable fields

Reply:
it can write, no more different